app/controllers/admin/routes.py

```diff
@@ -154,6 +154,11 @@ def eventDisplay(eventId):
         eventData['timeStart'] = event.timeStart.strftime("%-I:%M %p")
         eventData['timeEnd'] = event.timeEnd.strftime("%-I:%M %p")
         eventData["startDate"] = event.startDate.strftime("%m/%d/%Y")
+        # List below is to identify the last event in the series
```

Comment:
This comment says it is getting the last event, but it is really getting the next event